packages/trader/src/Assets/Trading/Categories/trade-categories.tsx

```diff
@@ -1,12 +1,23 @@
 import React from 'react';
 import { Text } from '@deriv/components';
+import { getLocalizedBasis } from '@deriv/shared';
 import { localize, Localize } from '@deriv/translations';
 import AccumulatorTradeDescription from './accumulator-trade-description';
 import { TurbosTradeDescription } from './turbos-trade-description';
 
 // Templates are from Binary 1.0, it should be checked if they need change or not and add all of trade types
 // TODO: refactor the rest of descriptions to use them as components like AccumulatorTradeDescription
-const TradeCategories = ({ category, onClick }: { category?: string; onClick: () => void }) => {
+type TTradeCategoriesProps = {
+    category?: string;
+    is_vanilla_fx?: boolean;
+    onClick: () => void;
+};
+
+const TradeCategories = ({ category, is_vanilla_fx, onClick }: TTradeCategoriesProps) => {
+    const vanilla_payout_text = is_vanilla_fx
```

Comment:
I cannot find the usage of this, where does this being used?

Reply:
It's not used anywhere, since I modified the logic in the file, not sure why it didn't trigger any alert